yaml/virtual-copy/virtual-copy-snap-vol.yml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: ci-primera3par-csp-secret
+  name: hpe3parprimera-secret
```

Comment:
Also for virtual copy some files have been removed. eg: virtual-copy-base-vol-tc4.yml. Had added then in my last PR. Not sure why they were removed.

Reply:
Addressed the comment and made appropriate changes. Please review.